client/a8c-for-agencies/sections/team/primary/team-list/columns.tsx

```diff
@@ -122,8 +167,25 @@ export const ActionColumn = ( {
 						label: translate( 'Delete user' ),
 						className: 'is-danger',
 						isEnabled: canRemove,
+						confirmationDialog: {
+							title: translate( 'Delete user' ),
+							children: translate( 'Are you sure you want to delete {{b}}%(memberName)s{{/b}}?', {
+								args: { memberName: member.displayName ?? member.email },
+								components: {
+									b: <b />,
+								},
+								comment: '%(memberName)s is the member name',
+							} ),
+							ctaLabel: translate( 'Delete user' ),
+							scary: true,
+						},
 					},
 			  ];
+	}, [ member, canRemove, translate ] );
+
+	if ( member.role === OWNER_ROLE ) {
```

Comment:
What exactly are we doing here? Could we leave a comment at the top of this code?

Reply:
I added a comment to explain why we are taking this action. For the owner, we restrict any action to prevent users from taking actions that might not work for the owners, such as deletion. 👍